docs/CHANGELOG.rst

```diff
@@ -20,6 +20,8 @@ Changed
   - ``workflow-bbduk-star-featurecounts-qc-paired``
   - ``workflow-bbduk-star-fc-quant-single``
   - ``workflow-bbduk-star-fc-quant-paired``
+- Save filtered count-matrix input file used in DESeq2 differential
```

Comment:
Consider referring to this file as an output file instead of input file, like [here](https://github.com/genialis/resolwe-bio/pull/761/files#diff-db40da7a4f13eac0b490a11805d2783cR52) because it appears in the outputs of the process. Perhaps  _Save filtered count-matrix output file produced by DESeq2 differential expression process_.